packages/owlbot-bootstrapper/common-container/utils.ts

```diff
@@ -18,13 +18,15 @@ import {uuid} from 'uuidv4';
 import {Octokit} from '@octokit/rest';
 import * as fs from 'fs';
 import {Language} from './interfaces';
+import {dir} from 'console';
 
-const BRANCH_NAME_FILE = 'branchName.md';
+export const BRANCH_NAME_FILE = 'branchName.md';
```

Comment:
Do we foresee more variables being passed between these containers? Perhaps creating a defined JSON or textproto formatted metadata file would be more robust than several individual files.